### PR TITLE
support logrotate for swss recording log

### PR DESCRIPTION
**Pull request by lguohan**

### CODE REVIEW

File: orchagent/orch.cpp [hunk 1]
```diff
@@ -242,6 +262,20 @@ void Orch::recordTuple(Consumer &consumer, KeyOpFieldsValuesTuple &tuple)
...
+    {
+        gLogRotate = false;
+
+        logfileReopen();
+
+        /* double check since reopen could fail */
+
+        if (gRecordOfs.is_open())
+        {
+            gRecordOfs << getTimestamp() << "|" << "#|logrotate on: " << gRecordFile << std::endl;
```
Comment (lguohan) [inline orchagent/orch.cpp]: shuotian, can you confirm if swssplayer supports this line or not.

Comment (stcheng) [inline orchagent/orch.cpp]: Not right now. I could add a code change after this commit.

**Comment by stcheng** [comment]
the duplicate logic could be moved to common library in the future.

File: orchagent/orch.cpp (hunk 1 above)
Comment (lguohan) [inline orchagent/orch.cpp]: ok. I removed the marker for now. Let's see what is the best way to add it later.

**Pull request merged**